simulation/src/range_likelihood.cpp

```diff
@@ -539,7 +539,7 @@ costFunction4(float ref_val,float depth_val)
   float proportion = 0.999f;
   float lhood = proportion + (1-proportion)*(top/bottom);
 
-  // safety fix thats seems to be required due to opengl ayschronizate
+  // safety fix that seems to be required due to opengl asyschronizate
```

Comment:
please check these fixes

Reply:
Darn, i left a typo in asyschronizate. But actually after more research (not sure I why I didn't do this the first time) I think the word should actually be `asynchronization`

Reply:
Can someone respond to this?

Reply:
I'm no native speaker, but [Wiktionary has such a word](https://en.wiktionary.org/wiki/asynchronization).

Reply:
Do you want to change this? Then we can merge.

Reply:
More importantly, who is Hordur? 😆 My guess would also go with asynchronization.